src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -157,6 +157,11 @@
     </StringProperty.ValueEditors>
   </StringProperty>
 
+  <StringProperty Name="PackageOutputPath"
+                  DisplayName="Package Output Path"
+                  Description="Determines the output path in which the packed package will be dropped."
+                  Category="General" />
```

Comment:
No help url available?

Reply:
I couldn't find a sensible URL for this property. There's a passing reference to it in a table of MSBuild properties that impact pack operations, but that table only mirrors the text we have here so doesn't add anything.